src/services/state-persistor/state-persistor-rules-write.helper.ts

```diff
@@ -20,6 +21,16 @@ export abstract class RulesWriteHelper extends StatePersistorRulesHelper {
     if (key === SP_KEY_LAYERS_OPEN) {
       return SP_AVAILABLE_STORAGES.localStorage
     }
+    if (
```

Comment:
Conditions could be added to if-clause above IMO. I'd also put the values into `const`s as others.